src/components/Form/Form.md

```diff
@@ -39,24 +38,31 @@ import { TitleLarge, TextInput, Spacer, Button, InfoMessage, ZipInput, ZipInputV
     },
     onSubmit: async (formData) => {
       await new Promise((resolve) => setTimeout(resolve, 500))
-      if (!!(Math.floor(Math.random() * 10) % 2)) {
-        throw new Error("Oh no, the api is broken (try again, it's random)")
+      if (count++ % 2 === 0) {
+        throw new Error("API Issue (Try again,, it alternates success & failure)")
       } else {
         alert(
           'form submission successful with values:' +
-            JSON.stringify(formData) +
-            "\n\nBut try again, it's random"
+            JSON.stringify(formData)
         )
       }
     },
   }}
 >
-  {(field, getFormErrorMessage, getFormIsValid) => (
+  {(field, getFormErrorMessage, getFormIsValid, getFormInteractedWith) => (
     <div>
       <TitleLarge.Serif.Book500>Example Form</TitleLarge.Serif.Book500>
 
       <Spacer.H16 />
 
+      {!!getFormInteractedWith() && (
```

Comment:
@iamvictorli just an example usage, but you will easily be able to call:
```js
      {!!getFormInteractedWith() && ( removeCTAButton() }
```

Reply:
and you probably don't need the `!!` part

Reply:
@roblevintennis You probably dont need the `!!` operator, since the getFormInteractedWith already returns a boolean